dist/validate.d.ts

```diff
@@ -6,5 +6,7 @@ export interface ValidationError {
     expected: any;
     error?: any;
 }
-export declare function request(compiledPath: CompiledPath | undefined, method: string, query?: any, body?: any, headers?: any): ValidationError[] | undefined;
+export declare function request(compiledPath: CompiledPath | undefined, pathParameters: {
```

Comment:
Can you revert all the dist files from the PR.  I'll recreate when I publish a new version.